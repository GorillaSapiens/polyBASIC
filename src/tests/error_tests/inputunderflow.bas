input a,b,c
rem assumes input from /dev/null
