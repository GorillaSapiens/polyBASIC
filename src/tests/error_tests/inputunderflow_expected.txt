a? SOURCE 1:1, DATA INPUT UNDERFLOW
